### DISCUSSION TITLE
Multi class tracking

### DISCUSSION THREAD

**Discussion by jnebrera** [tryolabs/norfair#270] [General] [answered]
Hi,

I just discovered your blog post on soccer tracking and hope it will help me do the same for basketball !!

One question, I want to track both players and the ball. Does Norfair support multi class (person / ball) tracking or do I need kind of two parallel instances (one for players, the other for the ball) to do so?

Thank you very much for this software. Regards

**Comment by jnebrera** [comment]
I reply myself.

From here, seems it can track multiple classes.

https://github.com/tryolabs/norfair/tree/master/demos/keypoints_bounding_boxes

**Comment by facundo-lezama** [comment, marked as answer]
Hi @jnebrera! Norfair allows for tracking multiple classes at the same time. The only thing you need to do is to specify the detection's class when building the `Detection` object. You can do so by using the `label` attribute.
